.github/dependabot.yml

```diff
@@ -10,148 +10,156 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
-      day: "sunday"
+      day: "thursday"
       time: "00:00"
-      timezone: "Asia/Kolkata"
+      timezone: "America/Chicago"
     allow:
       - dependency-type: "direct"
 
   - package-ecosystem: "cargo"
     directory: "/"
     schedule:
       interval: "weekly"
-      day: "sunday"
+      day: "thursday"
       time: "00:00"
-      timezone: "Asia/Kolkata"
+      timezone: "America/Chicago"
     allow:
       - dependency-type: "direct"
 
   - package-ecosystem: "composer"
     directory: "/"
     schedule:
       interval: "weekly"
-      day: "sunday"
+      day: "thursday"
       time: "00:00"
-      timezone: "Asia/Kolkata"
+      timezone: "America/Chicago"
     allow:
       - dependency-type: "direct"
 
   - package-ecosystem: "docker"
     directory: "/"
     schedule:
       interval: "weekly"
-      day: "sunday"
+      day: "thursday"
       time: "00:00"
-      timezone: "Asia/Kolkata"
+      timezone: "America/Chicago"
     allow:
       - dependency-type: "direct"
 
   - package-ecosystem: "mix"
     directory: "/"
     schedule:
       interval: "weekly"
-      day: "sunday"
+      day: "thursday"
       time: "00:00"
-      timezone: "Asia/Kolkata"
+      timezone: "America/Chicago"
     allow:
       - dependency-type: "direct"
 
   - package-ecosystem: "elm"
     directory: "/"
     schedule:
       interval: "weekly"
-      day: "sunday"
+      day: "thursday"
       time: "00:00"
-      timezone: "Asia/Kolkata"
+      timezone: "America/Chicago"
     allow:
       - dependency-type: "direct"
 
   - package-ecosystem: "gitsubmodule"
     directory: "/"
     schedule:
       interval: "weekly"
-      day: "sunday"
+      day: "thursday"
       time: "00:00"
-      timezone: "Asia/Kolkata"
+      timezone: "America/Chicago"
     allow:
       - dependency-type: "direct"
 
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
       interval: "weekly"
-      day: "sunday"
+      day: "thursday"
       time: "00:00"
-      timezone: "Asia/Kolkata"
+      timezone: "America/Chicago"
+    groups:
+      pipeline:
+        patterns:
+          - "*"
```

Comment:
perhaps we add this to the boilerplate

Reply:
Maybe we should have 2 options, and let them pick? Its hard to say what the right default here should be.